package.json

```diff
@@ -13,7 +13,7 @@
   "author": "EMBL-EBI",
   "license": "MIT",
   "scripts": {
-    "prepare": "cd ../.. && husky install src/ensembl/.husky",
+    "prepare": "husky install ./.husky",
```

Comment:
Try just `husky install`. I believe it will use the `.husky` directory by default.